plugins/palo_alto_cortex_xdr/help.md

```diff
@@ -661,45 +689,248 @@ Example output:
-|Manual Score|integer|False|Manual Score|
-|Manual Severity|string|False|Manual Severity|
-|Med Severity Alert Count|integer|False|Med Severity Alert Count|
-|Modification Time|integer|False|Modification Time|
+|Incident Name|string|False|Incident name|
+|Incident Sources|[]string|False|Incident sources|
+|Low Severity Alert Count|integer|False|Low severity alert count|
+|Manual Description|string|False|Manual description|
+|Manual Score|string|False|Manual score|
+|Manual Severity|string|False|Manual severity|
+|Med Severity Alert Count|integer|False|Med severity alert count|
+|Modification Time|integer|False|Modification time|
 |Notes|string|False|Notes|
-|Resolve Comment|string|False|Resolve Comment|
-|Rule Based Score|integer|False|Rule Based Score|
+|Resolve Comment|string|False|Resolve comment|
+|Rule Based Score|string|False|Rule based score|
 |Severity|string|False|Severity|
 |Starred|boolean|False|Starred|
 |Status|string|False|Status|
-|User Count|integer|False|User Count|
+|User Count|integer|False|User count|
 |Users|[]string|False|Users|
 |XDR URL|string|False|XDR URL|
 
+#### isolation_result
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Action ID|integer|False|Action ID|
+|Endpoints Count|integer|False|Endpoints count|
+|Status|string|False|Status|
+
+#### reply
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Number of Results|integer|False|Integer representing the number of results returned|
+|Query Cost|object|False|Float representing the number of query units collected for this API|
+|Remaining Quota|float|False|Float representing the number of query units available for you to use|
+|Results|xql_query_result|False|API results according to defined format field|
+|Status|string|False|API call status: 'SUCCESS', 'FAIL', 'PENDING', 'PARTIAL_SUCCESS'|
+
+#### xql_query_result
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Data|[]xql_query_result_data|False|List of obtained data results|
+|Event Subtype|string|False|String representing a unique ID of more than 1000 number of results|
+
+#### xql_query_result_data
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Product|string|False|Name of a product|
+|Time|integer|False|Integer representing the event time as epoch in ms|
+|Vendor Name|string|False|The name of a vendor|
+|Event ID|string|False|String representing the event identifier|
+|Event Subtype|string|False|String representing the subtype of an event|
+|Event Type|string|False|The type of event|
+|Insertion timestamp|integer|False|Integer representing insertion timestamp as epoch in ms|
+
 
 ## Troubleshooting
 
 * Isolate Endpoint fails with 500 error - This will happen if an isolation action (Isolate or Unisolate) is in progress on the selected endpoint. Wait a few minutes and try again. 
 
 # Version History
 
+* 3.0.0 - Add types to `Get XQL Query Results` | Add new trigger `Get Query Results` 
```

Comment:
I see that some types have been changed in incident which is causing backward compatibiilty. Are they necessary. If so can you please add the fields in the version history.